lib/galaxy/jobs/runners/util/job_script/CLUSTER_SLOTS_STATEMENT.sh

```diff
@@ -23,6 +23,9 @@ elif [ -f "$PBS_NODEFILE" ]; then
     GALAXY_SLOTS=`wc -l < $PBS_NODEFILE`
 elif [ -n "$LSB_DJOB_NUMPROC" ]; then
     GALAXY_SLOTS="$LSB_DJOB_NUMPROC"
+elif [ -n "$GALAXY_SLOTS" ]; then
+    # kubernetes runner injects GALAXY_SLOTS into environment
+    $GALAXY_SLOTS=$GALAXY_SLOTS
```

Comment:
This should be `GALAXY_SLOTS=$GALAXY_SLOTS`, you have an extra $.

Reply:
hah, good one. it still worked because we avoid the next case, I'll fix that in a rebase.